### PR TITLE
FIP: Filecoin Message Replay Protection

### PR DESCRIPTION
**Pull request by q9f**
Discussion to: #301 

Summary: Defines Filecoin chain identifiers and adds them to unsigned messages and signatures to prevent transactions to be replayed across different Filecoin networks.

_Note: I'm new to Filecoin and might have missed something in the specification. Feel free to point to everything I should now._

### CODE REVIEW

**Comment by jleni** [comment]
included my comment in https://github.com/filecoin-project/FIPs/discussions/301

File: FIPS/fip-9999-replay-protection.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+## Security Considerations
+This proposal supposedly enhances chain and user security by preventing replay protection across different Filecoin networks.
+
+Notably, EIP-155 has been in use by Ethereum for more than five years and has proven robust.
+
+## Incentive Considerations
+_Not applicable._
+
+## Product Considerations
+_Not applicable._
```
Comment (anorth) [inline FIPS/fip-9999-replay-protection.md:100]: I'd say there are applicable considerations here, notably considering the product implications of breaking compatibility in all existing tools and applications that sign Filecoin messages.

Comment (q9f) [reply FIPS/fip-9999-replay-protection.md:100]: You are probably right; I may have misunderstood the term _product_ here. I'll give this some thought.

**Comment by kaitlin-beegle** [comment]
Hey @q9f! Perhaps I missed the thread- can I ask why you closed this FIP? Do you no longer plan to work on it?

**Comment by q9f** [comment]
It hasn't been accepted as a draft and I considered it no longer desired.

**Comment by kaitlin-beegle** [comment]
@q9f as long as you don't disagree, I'm reopening this issue. It simply hadn't been merged because FIP editors hadn't been able to provide feedback yet.  

Oftentimes, PRs stay open until multiple rounds of revision have taken place.  However, FIP editors have all been pretty heads-down on the FVM FIPs. Replay protection may not be an immediate priority, but it's still important.

In general, we are quite capacity limited, and governance topics intentionally move quite slowly.  If you have questions or want to push on us, please do so! But no need to close issues and PRs- if the idea wasn't viable, you would know.

**Comment by q9f** [comment]
Ok, that's fine. Maybe, I just misunderstood the process. I thought assigning a number and merging a draft is a no-brainer.

**Review by kaitlin-beegle — APPROVED** [review]

**Pull request merged**